### PR TITLE
Extend clearfix via placeholder, not class

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Changed clearfix so it can be extended via a placeholder as the linter dictates.
- Replaced any examples where clearfix is extended via class with placeholder

## QA

- Check code still builds
- QA with vf.io

### CODE REVIEW

File: scss/_layout_clearfix.scss [hunk 1]
```diff
@@ -1,9 +1,13 @@
 // Force a parent to clear floated children
 
 @mixin vf-clearfix {
-  .clearfix::after {
+  %clearfix::after {
```
Comment (pmahnke) [inline scss/_layout_clearfix.scss]: should this just be %clearfix?

File: scss/_layout_clearfix.scss [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
     content: '';
     clear: both;
     display: block;
   }
+
+  %clearfix::after {
```
Comment (pmahnke) [inline scss/_layout_clearfix.scss]: should this be .clearfix::after?

File: scss/_layout_clearfix.scss (hunk 1 above)
Comment (anthonydillon) [inline scss/_layout_clearfix.scss]: This should be:

```scss
%clearfix::after {
    content: '';
    clear: both;
    display: block;
  }

  .clearfix {
    @extend %clearfix;
  }
```

@barrymcgee ???

Comment (barrymcgee) [inline scss/_layout_clearfix.scss]: Yes, sorry - would seem my quick find and replace to borked this..

**Comment by barrymcgee** [comment]
@pmahnke No, `/docs/utilities/__floats.md` is correct

**Comment by pmahnke** [comment]
@barrymcgee assuming the CI completes, LGTM 👍

**Pull request merged**